swift/apple/FirezoneKit/Sources/FirezoneKit/Helpers/LogExporter.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  LogExporter.swift
+//  (c) 2024 Firezone, Inc.
+//  LICENSE: Apache-2.0
+//
+
+import AppleArchive
+import Foundation
+import System
+
+/// Convenience module for smoothing over the differences between exporting logs on macOS and iOS.
+///
+/// On macOS, we must export the app's log dir and tunnel's log dir separately to temp files, and then join
+/// these into a final compressed archive saved at the user's specified location. This is because Apple Archive
+/// compression APIs do not easily allow compressing from multiple disparate paths in one go; only a single
+/// source directory is supported.
+///
+/// On iOS, the app process can compress the entire log directory itself, with no help from the tunnel process,
+/// thus avoiding IPC. In this case we write directly to the provided archiveURL.
+struct LogExporter {
+  enum ExportError: Error {
+    case sourceDirectoryInvalid
+    case invalidArchiveURL
+    case invalidFileHandle
+  }
+
+  private static let fileManager = FileManager.default
+
+  static func export(to archiveURL: URL) async throws {
```

Comment:
If all functions on the class are `static`, there is no need for a class, is there?